src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -300,12 +334,19 @@ private static T PromptProperty<T>(object model, T property, string memberName,
                     .MakeGenericMethod(property.GetType());
 
                 return (T)generic.Invoke(null, new object[] { message, property.GetType().GetEnumValues(), null, property, null });
-            }
-            else if (typeof(T) != typeof(string) && typeof(IEnumerable<string>).IsAssignableFrom(typeof(T)))
+            }
+            else if (typeof(IEnumerable<string>).IsAssignableFrom(typeof(T)) || typeof(IEnumerable<string>).IsAssignableFrom(typeFromModel))
             {
-                // If property is an IEnumerable<string>, we take in a comma-delimited string, and validate each split item, then return the split array
-                string promptResult = Prompt.Input<string>(message, null, new[] { FieldValidation.ValidateProperty(model, memberName, property) });
-                return (T)(object)promptResult?.Split(',', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
+                string combinedString = null;
+
+                if (property is List<string> propList && propList.Any())

```

Comment:
>List [](http://example.com/codeflow?start=32&length=4)

IEnumerable<string>

Reply:
Actually, isn't this first clause unnecessary, since we already know it's an IEnumerable<string> ?

Reply:
Changed to IEnumerable